en/components/themes/palette.md

```diff
@@ -27,10 +27,10 @@ $my-color-palette: igx-palette(
 );
 ```
 
-Now we have a palette, that contains exactly 42 colors! Whoa, wait, what? How did that happen? You provided 2 and got 42? Where did the other 40 colors come from?
+Now we have a palette that contains exactly 43 colors! Whoa, wait, what? How did that happen? You provided 2 and got 43? Where did the other 41 colors come from?
 Let's stop here to explain what just happened as it's quite important. When you provided `primary` and `secondary` colors, we took those and generated shades and accent colors for each one. Basically now in your palette you have 2 sub-palettes for `primary` and `secondary` colors. Each sub-palette contains 13 additional color variations based on the original color. 5 of the 13 colors are lighter shades of your original color, and 4 are darker. The remaining 4 colors are more exaggerated 'accent' versions of the original color. With the original color that makes for a total of 14 colors in each palette.
 
-With so many colors in each sub-palette you may be wondering, how on Earth will I know which one is which, right? It's quite simple, really. Each of the colors in the sub-palette has a number. We assign the number `500` to the original color. The lighter shades start from `50` and range to `400`. The darker shades start from `600` and range to `900`. The accent colors have string names `A100`, `A200`, `A400`, and `A700`, respectively. Okay, but now that's only 28 out of 42. Don't worry, there's another sub-palette we give you. One that consists of gray 'colors', called `grays`. It's just like the other two color sub-palettes, but doesn't include any accent variations. Good, now we're up to 28 + 10 for a total of 38 colors. That is still a some way from 42. Where do the other 4 colors come from? Let's solve the final puzzle. Remember you can also have 4 additional colors for `info`, `success`, `warn` and `error`.
+With so many colors in each sub-palette you may be wondering, how on Earth will I know which one is which, right? It's quite simple, really. Each of the colors in the sub-palette has a number. We assign the number `500` to the original color. The lighter shades start from `50` and range to `400`. The darker shades start from `600` and range to `900`. The accent colors have string names `A100`, `A200`, `A400`, and `A700`, respectively. Okay, but now that's only 28 out of 4e. Don't worry, there's another sub-palette we give you. One that consists of gray 'colors', called `grays`. It's just like the other two color sub-palettes, but doesn't include any accent variations. Good, now we're up to 28 + 10 for a total of 38 colors. That is still a some way from 43. Where do the other 5 colors come from? Let's solve the final puzzle. Remember you can also have 5 additional colors for `surface`, `info`, `success`, `warn` and `error`.
```

Comment:
Is **4e** correct in the sentence "Okay, but now that's only 28 out of **4e**."? Should it be 43 instead?

Reply:
Absolutely not :) Thanks for spotting the error.